### PR TITLE
Header and Section branch

### PR DESCRIPTION
**Pull request by ameerhamzakhalid**
I created the whole node module and hide it with gitignore
I followed template 3
I created the HTML file
I created the CSS file
I added img folder
I added images in html and css file
I created mobile version and desktop version of template
lastly, I edited the readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                <li>Contact</li>
+            </ul>
+        </nav>
+        <div class="container container-flex">
+            <h1 class="header-title">
+                Hey there. I'm Ameer Hamza
+            </h1>
+            <h1>
+                <span>I'm a software developer</span>
+            </h1>
```
Comment (mahbubul14) [inline index.html]: - [x] Please keep one `h1` tag for your page. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
-# portfolio-ameer-hamza
\ No newline at end of file
+---
+layout: default
+---
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+
```
Comment (mahbubul14) [inline README.md]: - [x] You should add a title related to your project and a short description of it. Kindly work on that. :writing_hand:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @ameerhamzakhalid ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [x] Pull Request should have a descriptive title telling you what changes are inside. Currently, your PR title is not related to your project. Please work on it.
p1

- [x] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Clean code.

### Optional suggestions

- [x] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices, ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @ameerhamzakhalid , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**